tiatoolbox/utils/visualization.py

```diff
@@ -351,20 +381,26 @@ def plot_graph(
     Drawing a graph onto a canvas.
 
     Args:
-        canvas (np.ndarray): Canvas to be drawn upon.
-        nodes (np.ndarray): List of nodes, expected to be Nx2 where
-            N is the number of nodes. Each node is expected to be of
-            `(x, y)` and should be within the height and width of the
-            canvas.
-        edges (np.ndarray): List of egdes, expected to be Mx2 where
-            M is the number of edges. Each edge is defined as `(src, dst)`
-            where each is respectively the index of within `nodes`.
-        node_colors (tuple or np.ndarray): A color or list of node colors.
-            Each color is expected to be `(r, g, b)` and is between 0-255.
-        edge_colors (tuple or np.ndarray): A color or list of node colors.
-            Each color is expected to be `(r, g, b)` and is between 0-255.
-        node_size (int): Radius of each node.
-        edge_size (int): Linewidth of the edge.
+        canvas (np.ndarray):
+            Canvas to be drawn upon.
+        nodes (np.ndarray):
+            List of nodes, expected to be Nx2 where N is the number of
+            nodes. Each node is expected to be of `(x, y)` and should be
+            within the height and width of the canvas.
+        edges (np.ndarray):
+            List of edges, expected to be Mx2 where M is the number of
+            edges. Each edge is defined as `(src, dst)` where each is
```

Comment:
"Each edge is defined as `(src, dst)` where each is respectively the index of within `nodes`."
This doesn't make sense to me. Could we rephrase it?